### DISCUSSION TITLE
[Frontend] Remix.run

### DISCUSSION THREAD

**Discussion by BenoitAverty** [realworld-apps/realworld#653] [WIP Implementations]
Hello,

Remix is a paid server-side rendering react framework that's currently in beta (https://remix.run/features).

I've started implementing the Conduit frontend using this technology as an example app, so that anyone can see how to code using this framework without buying a license.

The repo is here : https://github.com/BenoitAverty/realworld-remix.run

It isn't finished yet, some features are missing. But the architecture is stable and shouldn't change much.

Help is welcome to bring it over the finish line, and any feedback is appreciated.

**Comment by geromegrignon** [comment]
Hello @BenoitAverty,

I can't wait to attend your talk about Remix at Devfest Nantes (France) next month!
I plan to create accounts on social media platforms for RealWorld to promote completed and WIP implementations.

**Comment by MatiasVerdier** [comment]
@BenoitAverty Are you still planning to work on this now that Remix it's open source and has reached v1.0?

**Comment by mdrahiem** [comment]
Hello @BenoitAverty I see the repo is pretty outdated and I would love to contribute to the repo. Please let me know if it's more feasible to create a PR for this repo or to create a new repo for the new implementation with latest version of Remix
@geromegrignon

**Reply by geromegrignon** [reply]
Hello @mdrahiem, I encourage you to create your own repo as we didn't have news from Benoit for a while.
I'll be happy to add it on CodebaseShow once completed!

**Reply by mdrahiem** [reply]
Hey @geromegrignon, thanks for your quick reply. I did forked the starter kit in this [repo](https://github.com/mdrahiem/remix-realworld-starter-kit). Looking forward to implement with best practices 🙂
